setup.cfg

```diff
@@ -1,9 +1,89 @@
 [metadata]
-description-file = README.md
+name = ewatercycle
+version = 1.0.0
+description = A Python package for running and validating a hydrology model
+author = Stefan Verhoeven
+author_email = s.verhoeven@esciencecenter.nl
+url = https://www.ewatercycle.org/
+license = Apache Software License 2.0
+keywords =
+    ewatercycle
+    FAIR
+    BMI
+    Geoscience
+classifiers =
+    Development Status :: 2 - Pre-Alpha
+    Intended Audience :: Developers
+    License :: OSI Approved :: Apache Software License
+    Natural Language :: English
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: GIS
+    Topic :: Scientific/Engineering :: Hydrology
+    Typing :: Typed
+project_urls =
+    Bug Tracker = https://github.com/eWaterCycle/ewatercycle/issues
+    Documentation = https://ewatercycle.readthedocs.io/
+    Source Code = https://github.com/eWaterCycle/ewatercycle
+
+[options]
+zip_safe = False
+python_requires = >=3.7
+packages = find:
```

Comment:
Is there any documentation on how this works?

Reply:
See https://setuptools.readthedocs.io/en/latest/userguide/package_discovery.html